code/_onclick/hud/human.dm

```diff
@@ -56,7 +56,7 @@
 
 	using = new /obj/screen/mov_intent()
 	using.icon = ui_style
-	using.icon_state = (mymob.m_intent == "run" ? "running" : "walking")
+	using.icon_state = (mymob.m_intent == "run" ? "running" : ("walk" ? "walking" : "sprinting"))

```

Comment:
Why aren't these icon states just the same as the goddamn variable to begin with.

What the FUCK